package.json

```diff
@@ -72,6 +72,7 @@
     "url": "https://github.com/Leonidas-from-XIV/node-xml2js.git"
   },
   "dependencies": {
+    "promise-polyfill": "7.1.2",
```

Comment:
As I see it Node has had promises since version 0.12 (which we stopped supporting a while ago), so I think I would prefer not to add polyfills.

Reply:
Some people might use xml2js in a browser context (I do), in which case the polyfill is useful, isn't it?

Reply:
In this case I would prefer people to use browserification tools. After all, this is called `node-xml2js` and no testing on browsers is done so I believe we should either support browsers or not add dependencies "just in case", without properly testing on browsers anyway.